src/main/resources/db/migration/V1.15__drop_location_name_constraint.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE location
```

Comment:
I think it's still worth having these constraints, as although the code is there to prevent this, it could still happen if two users were uploading the locations at the same time

I also think, if we were to keep these, that this overall method should be marked as Transactional so it rolls back all changes, rather than saving some and then stopping when it fails on this

Reply:
Removed the flyway script to drop constraint